src/survey.ts

```diff
@@ -581,102 +587,72 @@ export class SurveyModel extends SurveyElementCore
 
   /**
-   * An event that is raised before rendering "Remove" button for removing a row from Matrix Dynamic question.
-   * @see QuestionMatrixDynamicModel
+   * An event that is raised before rendering the Remove button in a row of a [Dynamic Matrix](https://surveyjs.io/form-library/examples/questiontype-matrixdynamic/). Use this event to hide the Remove button for individual matrix rows.
    * @see onMatrixRowRemoving
    * @see onMatrixRowRemoved
    */
   public onMatrixAllowRemoveRow: EventBase<SurveyModel, MatrixAllowRemoveRowEvent> = this.addEvent<SurveyModel, MatrixAllowRemoveRowEvent>();
 
   /**
-   * An event that is raised before creating cell question in the matrix. You can change the cell question type by setting different options.cellType.
-   * @see onMatrixBeforeRowAdded
-   * @see onMatrixCellCreated
-   * @see QuestionMatrixDynamicModel
-   * @see QuestionMatrixDropdownModel
+   * An event that is raised before a cell in a [Multi-Select Matrix](https://surveyjs.io/form-library/examples/questiontype-matrixdropdown/) or [Dynamic Matrix](https://surveyjs.io/form-library/examples/questiontype-matrixdynamic/) is created. Use this event to change the type of individual matrix cells.
+   * @see onMatrixAfterCellRender
    */
   public onMatrixCellCreating: EventBase<SurveyModel, MatrixCellCreatingEvent> = this.addEvent<SurveyModel, MatrixCellCreatingEvent>();
 
   /**
-    * An event that is raised for every cell created in Matrix Dynamic and Matrix Dropdown questions.
-    * @see onMatrixBeforeRowAdded
-    * @see onMatrixCellCreating
-    * @see onMatrixRowAdded
-    * @see QuestionMatrixDynamicModel
-    * @see QuestionMatrixDropdownModel
+    * An event that is raised after a cell in a [Multi-Select Matrix](https://surveyjs.io/form-library/examples/questiontype-matrixdropdown/) or [Dynamic Matrix](https://surveyjs.io/form-library/examples/questiontype-matrixdynamic/) is created.
+    * @see onMatrixAfterCellRender
     */
   public onMatrixCellCreated: EventBase<SurveyModel, MatrixCellCreatedEvent> = this.addEvent<SurveyModel, MatrixCellCreatedEvent>();
 
   /**
-   * An event that is raised for every cell after is has been rendered in DOM.
+   * An event that is raised for every matrix cell after it is rendered to the DOM.
    * @see onMatrixCellCreated
-   * @see QuestionMatrixDynamicModel
-   * @see QuestionMatrixDropdownModel
    */
   public onMatrixAfterCellRender: EventBase<SurveyModel, MatrixAfterCellRenderEvent> = this.addEvent<SurveyModel, MatrixAfterCellRenderEvent>();
 
   /**
-   * An event that is raised when cell value is changed in Matrix Dynamic and Matrix Dropdown questions.
-   * @see onMatrixCellValueChanging
+   * An event that is raised after a cell value is changed a [Multi-Select Matrix](https://surveyjs.io/form-library/examples/questiontype-matrixdropdown/) or [Dynamic Matrix](https://surveyjs.io/form-library/examples/questiontype-matrixdynamic/).
```

Comment:
An event that is raised after a cell value is changed in a [Multi-Select Matrix](https://surveyjs.io/form-library/examples/questiontype-matrixdropdown/) or [Dynamic Matrix](https://surveyjs.io/form-library/examples/questiontype-matrixdynamic/).